### ISSUE TITLE
[Solved] Getting 422 (Unprocessable Entity) on trying to create or update customer metafield

### ISSUE THREAD

**Issue by ADTC** [MONEI/Shopify-api-node#516]
I'm attempting to **create or update** customer metafields. Please let me know what I could be doing wrong:
```js
const Shopify = require('shopify-api-node');

const shopify = new Shopify({
  shopName: myShopifyDomain,
  apiKey: api_key,
  password: password
});

await shopify.customer.update(customer_id, {
  id: customer_id,
  metafields: [{
    key: 'my_key_1',
    value: 'my_value_1',
    value_type: 'string',
    namespace: 'my_ns_1'
  }, {
    key: 'my_key_2',
    value: 'my_value_2',
    value_type: 'string',
    namespace: 'my_ns_2'
  }]
});
```
I receive `HTTPError: Response code 422 (Unprocessable Entity)`. BTW, if it's important, I'm using this module in AWS Lambda.

**Comment by lpinca** [comment]
Thank you the suggestion but I think it is better to rely on official examples from Shopify API documentation and Shopify Community. This library is only a user friendly wrapper.

Feel free to open a PR to document how to read error details (when available).

**Comment by ADTC** [comment]
@lpinca true, but I was referring more about having a complete set of examples on how to use your project (in code) in as many scenarios as possible, so that someone new can get started quicker than I did.

I forked your project and ran `jsdoc` on it. Published here (with the config file and docs folder in my jsdoc branch): https://adtc.github.io/Shopify-api-node/

Of course, this is only autogenerated from all the jsdoc comments you left in the code (thankfully), but it might be a good starting point to improve those comments and even tap into the ability of jsdoc to include examples automatically.

Thank you for your reply btw.
